src/Sparrow/Json/JsonContextPoolBase.cs

```diff
@@ -270,6 +270,10 @@ private void Cleanup(object _)
                         _globalStack.Push(context);
                 }
             }
+            catch (OutOfMemoryException)
```

Comment:
What if we get `EarlyOutOfMemoryException`?

Reply:
As far as I can tell we can get EOOM only when we allocating. Maybe I missing something but I don't see a code path here that might throw it.